wagtail/admin/templates/wagtailadmin/shared/pagination_nav.html

```diff
@@ -9,7 +9,49 @@
 {% resolve_url linkurl as url_path %}
 
 <nav class="pagination" aria-label="{% trans 'Pagination' %}">
-    <p>{% blocktrans trimmed with page_num=items.number total_pages=items.paginator.num_pages %}Page {{ page_num }} of {{ total_pages }}.{% endblocktrans %}</p>
+    <div>
+        <div class="page-size-selector">
+            <label for="page-size-select">Items per page:</label>
+            <select id="page-size-select" aria-label="Select page size">
```

Comment:
This label needs to be translateable, and we don’t want an `aria-label` since there is one on the page already. We’ll also need to refactor this to use `rawformattedfield` to get the expected styling of the field.